### PR TITLE
[stable/rabbitmq] allow set extra securityContext configuration

### PR DESCRIPTION
**Pull request by f84anton**
#### What this PR does / why we need it: 
Allow to set extra securityContext. For example, manage capabilities or pod sysctls:
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.15/#securitycontext-v1-core

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [X] Chart Version bumped
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/rabbitmq/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: rabbitmq
-version: 6.5.0
+version: 6.5.1
```
Comment (carrodher) [inline stable/rabbitmq/Chart.yaml]:
```suggestion
version: 6.6.0
```

**Review by carrodher — COMMENTED** [review]
Thanks for the contribution, before continue, I think in this case you should increment the chart *minor* version instead of the patch one, according to [semver](http://semver.org/):

> Given a version number `MAJOR.MINOR.PATCH`, increment the:
> - MAJOR version when you make incompatible API changes,
> - MINOR version when you add functionality in a backwards-compatible manner, and
> - PATCH version when you make backwards-compatible bug fixes.
> Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format.

**Comment by f84anton** [comment]
@carrodher, I agree. Thank you.

**Pull request merged**